### DISCUSSION TITLE
How set the video fps on infered mode?

### DISCUSSION THREAD

**Discussion by info2000** [Megvii-BaseDetection/YOLOX#1374] [Q&A]
I had videos at 25 frames per second but want predict only on 5 frames per second.
How can set this without re-encode the video?

thanks

**Comment by Razamalik4497** [comment]
why can't you skip every 4 frames & make prediction on 5th 

it will help you :
https://stackoverflow.com/questions/39622281/capture-one-frame-from-a-video-file-after-every-10-seconds
